packages/ag-charts-community/src/chart/interaction/interactionManager.ts

```diff
@@ -167,6 +190,37 @@ export class InteractionManager extends BaseManager<InteractionTypes, Interactio
         });
     }
 
+    private makeRegion(name: RegionName): InteractionRegion {
+        return {
+            name,
+            listeners: new Listeners<InteractionTypes, (event: InteractionEvent<InteractionTypes>) => void>(),
+        };
+    }
+
+    public addRegion(name: RegionName, node: BBoxProvider): ObservableRegion {
+        const region = this.makeRegion(name);
+        const interactionManager = this;
+        this.regions.add(region, node);
```

Comment:
I don't think that the region manager needs to worry about that because `node: BBoxProvider` is just an interface for a scene `Node`. It accesses the bounding box through `computeBBox()` and the scene-graph manages update/caching of these boxes based on the nodes' dirty states.